vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java

```diff
@@ -1426,6 +1429,12 @@ private void visit(HttpRequest request, VisitorParameters parameters, boolean st
 
     // ------------------------------------------------ Helpers ------------------------------------------------
 
+    private static long doomMillis(HttpRequest request) {
+        long connectedAtMillis = request.getConnectedAt(MILLISECONDS);
```

Comment:
FYI `HttpRequest.getConnectedAt` returns the timestamp when the TLS/HTTP connection was initiated, not when the request was received. `HttpRequest.relativeCreatedAtNanoTime()` returns the time the request was handed over from Jetty.

Reply:
Hmm, how's that for HTTP/2 with multiple requests per connection?

I guess we could use the JDisc creationTime, but in one of my dumps, the difference between the two was 13s, and in the other it was 2s and 0.14s, all of which I think are a bit high.

Reply:
... well, if the connection had several requests, I guess that could explain the difference.